fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -873,6 +875,8 @@ nonReserved
     | DATETIME
     | DATETIMEV2
     | DATEV2
+    | DATETIMEV1
+    | DATEV1
```

Comment:
nereids should update LogicalPlanBuilder and other files too.
please add create table and query test for legacy planner and Nereids